src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -3309,7 +3309,7 @@ public object Convert(object value, Type targetType, object parameter, System.Gl
                         return resourceDictionary["PrimaryCharcoal200Brush"] as SolidColorBrush;
                 };
             }
-            return resourceDictionary["PrimaryCharcoal200Brush"] as SolidColorBrush;
```

Comment:
hmm why the default is color for boolean?

Reply:
This is for 'null' objects. PrimaryCharcoal200Brush is for default.